sauna/plugins/ext/network.py

```diff
@@ -0,0 +1,130 @@
+import time
+
+from sauna.plugins.base import PsutilPlugin
+from sauna.plugins import PluginRegister
+
+my_plugin = PluginRegister('Network')
+
+
+@my_plugin.plugin()
+class Network(PsutilPlugin):
+
+    def __init__(self, config):
+        super().__init__(config)
+
+    @my_plugin.check()
+    def upload_kbytes_speed(self, check_config):
```

Comment:
I'm not sure about this `kbytes` thing. Could you check if it's difficult to let the user use its own unit? I think the memory plugin do that already.